lib/commons/standards/implicit-html-roles.js

```diff
@@ -109,7 +109,7 @@ const implicitHtmlRoles = {
 				listElement && listElement.nodeName.toLowerCase() === 'datalist';
 		}
 
-		switch ((vNode.attr('type') || '').toLowerCase()) {
+		switch ((vNode.props.type || '').toLowerCase()) {
```

Comment:
props.type is already converted to the appropriate case.

```suggestion
		switch (vNode.props.type) {
```

Reply:
Thanks @WilcoFiers.